modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java

```diff
@@ -140,11 +141,30 @@ private void _fillTemplateContextTemplateVariables(
 		TemplateVariableGroup generalVariablesTemplateVariableGroup =
 			templateVariableGroupsMap.get("general-variables");
 
+		TemplateVariableGroup copyGeneralVariablesTemplateVariableGroup =
```

Comment:
Hi @victorKammerer , I checked in the portal with `git grep "new TemplateVariableGroup"` and we always initialize this inside of a class `TemplateHander` on this class we can also set the restricted variable.

Reply:
Done, @linolaoj! I created and implemented a new `NotificationTemplatesTemplateHandler` class